src/main/java/com/ays/user/service/impl/UserServiceImpl.java

```diff
@@ -63,6 +90,13 @@ public void updateUser(final String id, final UserUpdateRequest updateRequest) {
         userRepository.save(userEntity);
     }
 
+    /**
+     * Deletes a user with the specified ID.
+     *
+     * @param id the ID of the user to delete
+     * @throws AysUserNotExistByIdException   if the user does not exist
+     * @throws AysUserAlreadyDeletedException if the user is already deleted
+     */
     @Override
     public void deleteUser(final String id) {
 
```

Comment:
Bu sınıf için ben Java DOC eklemiştim, buradan kaldırabiliriz yoksa çakışacak

Reply:
Bu sınıftaki tüm Java Doc'ları kaldırdım.